plugin.video.embycon/resources/lib/play_utils.py

```diff
@@ -356,37 +356,59 @@ def playFile(play_info, monitor):
     list_item.setPath(playurl)
     list_item = setListItemProps(id, list_item, result, server, listitem_props, item_title)
 
+    player = xbmc.Player()
     playlist = xbmc.PlayList(xbmc.PLAYLIST_VIDEO)
     playlist.clear()
     playlist.add(playurl, list_item)
-    xbmc.Player().play(playlist)
+    player.play(playlist)
+
+    if seekTime != 0:
+        player.pause()
+        monitor = xbmc.Monitor()
+        count = 0
+        while not player.isPlaying() and not monitor.abortRequested() and count != 100:
+            count = count + 1
+            xbmc.sleep(100)
```

Comment:
you might want to use waitforabort(1) here and use 10 in the count variable

Reply:
yeah I did try that but there was a very noticeable delay from when the play stared to when the seek happened. This approach worked better in testing on various hardware.